packages/components/Notification/src/Notification.styling.ts

```diff
@@ -49,6 +58,22 @@ export const stylingSettings: UseStylingOptions<NotificationProps, NotificationS
       },
       ['color'],
     ),
+    message: buildProps(
+      (tokens: NotificationTokens) => {
+        return {
+          style: {
+            color: tokens.color,
+            fontSize: tokens.fontSize,
+            fontWeight: tokens.fontWeight,
+            lineHeight: tokens.fontLineHeight,
+            letterSpacing: tokens.fontLetterSpacing,
+            flex: 1,
+            flexGrow: 1,
+          },
+        };
+      },
+      ['color'],
+    ) as TextStyle,
```

Comment:
Is this an ok way to do it? I added this to address the { ... } is not assignable to type 'TextStyle' error

Reply:
What's the full error?

Reply:
The types of 'style' are incompatible between these types.

Type '{ color: ColorValue; fontSize: number | keyof FontSizes; fontWeight: "bold" | "600" | keyof FontWeights | "normal" | "100" | "200" | "300" | ... 4 more ... | "900"; lineHeight: number; letterSpacing: number; flex: number; flexGrow: number; }' is not assignable to type 'StyleProp<TextStyle>'.

Type '{ color: ColorValue; fontSize: number | keyof FontSizes; fontWeight: "bold" | "600" | keyof FontWeights | "normal" | "100" | "200" | "300" | ... 4 more ... | "900"; lineHeight: number; letterSpacing: number; flex: number; flexGrow: number; }' is not assignable to type 'TextStyle'.

Reply:
Talked and solved offline. Solution is to use `...fontStyles.from(tokens, theme)`